pkg/apis/monitoring/v1alpha1/blackbox_exporter_types.go

```diff
@@ -0,0 +1,38 @@
+package v1alpha1
+
+import (
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type BlackboxExporter struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec BlackboxExporterSpec `json:"spec"`
+}
+
+type BlackboxExporterSpec struct {
+	// Define resources requests and limits for each blackbox-exporter pod.
+	Resources v1.ResourceRequirements `json:"resources,omitempty"`
+	// Namespace for a blackbox-exporter deployment.
+	Namespace string `json:"namespace,omitempty"`
+	// Image to use for a blackbox-exporter deployment.
+	Image string `json:"image,omitempty"`
+	// Version string used for labeling objects
+	Version string `json:"version,omitempty"`
+	// Image of config-reloader sidecar container
+	// +kubebuilder:default="jimmidyson/configmap-reload:v0.5.0"
+	ConfigmapReloaderImage string `json:"configmapReloaderImage,omitempty"`
+	// Secure listening port
+	// +kubebuilder:default=19115
+	Port int `json:"port,omitempty"`
+	// Insecure listening port
+	// +kubebuilder:default=9115
+	InternalPort int `json:"internalPort,omitempty"` // TODO: we should probably remove this
```

Comment:
Then let's remove it now and wait for people to explicitly ask for it and the reason why. :)

Reply:
Makes sense. I removed it. :)